github-filter-worker/src/index.ts

```diff
@@ -11,13 +11,20 @@ const hcConfig: Config = {
     'exception': 1
   }
 }
+const emojiRegex = new RegExp(":([a-zA-Z0-9-_])+?:")
 
 const listener = hc(hcConfig, event => {
   event.respondWith(handleRequest(event.request))
 })
 
 addEventListener('fetch', listener)
 
+// Try to lookup the emoji through the KV namespace, return the name if not found
+async function lookupEmoji(name: string): string {
+  let value = await emojis.get(name)
+  return value === null ? value : name
```

Comment:
`null` is falsey, so explicit comparison is not required:
```js
> let val = null;
undefined
> val ? "a" : "b"
'b'
> val = "abcdef"
'abcdef'
> val ? "a" : "b"
'a'
```
```suggestion
  return value ? value : name
```